client/dead.lua

```diff
@@ -98,7 +97,8 @@ local function logDeath(victim, attacker, weapon)
     local killerName = GetPlayerName(killerId) .. ' ' .. '(' .. GetPlayerServerId(killerId) .. ')' or Lang:t('info.self_death')
     local weaponLabel = WEAPONS[weapon].label or 'Unknown'
     local weaponName = WEAPONS[weapon].name or 'Unknown'
-    TriggerServerEvent('qb-log:server:CreateLog', 'death', Lang:t('logs.death_log_title', { playername = playerName, playerid = GetPlayerServerId(playerId) }), 'red', Lang:t('logs.death_log_message', { killername = killerName, playername = playerName, weaponlabel = weaponLabel, weaponname = weaponName }))
+    local message = Lang:t('logs.death_log_message', { killername = killerName, playername = playerName, weaponlabel = weaponLabel, weaponname = weaponName })
+    lib.callback('qbx_medical:server:logDeath', false, false, message)
```

Comment:
Can just change it to await, no need for a vscode extension warning

Reply:
What do you mean by that? Why should it be an await over a callback? And what is this warning you're talking about?

Reply:
If you send false to the argument which should be the callback function you get a warning in vscode for wrong data type, while not wrong, it's annoying

Reply:
I get no warning regarding that when using the overextended lua extension and the sumneko lua extension and it doesnt error anything out in FiveM?